scripts/devops_tasks/setup_execute_tests.py

```diff
@@ -22,12 +22,13 @@
 # evaluating whether we want this or not.
 ALLOWED_RETURN_CODES = [] 
 
-def prep_and_run_tests(targeted_packages, python_version):
+def prep_and_run_tests(targeted_packages, python_version, testRes):
     print('running test setup for {}'.format(targeted_packages))
     run_check_call([python_version, dev_setup_script_location, '-p', ','.join([os.path.basename(package_path) for package_path in targeted_packages])], root_dir)
 
     print('Setup complete. Running pytest for {}'.format(targeted_packages))
-    command_array = [python_version, '-m', 'pytest']
+    command_array = [python_version, '-m', 'pytest','--doctest-modules']
```

Comment:
What does the `--doctest-modules` do? Is it required always? Can you add it to to the `testRes` variable by default?

Reply:
Spacing too.